docs/tables/aws_route53_resolver_endpoint.md

```diff
@@ -0,0 +1,79 @@
+# Table: aws_route53_resolver_endpoint
+
+The AWS Route 53 Resolver endpoints in a Virtual Private Cloud (VPC) that is used for DNS management.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+	name,
+	id,
+	direction,
+	ip_address_count,
+	status
+from
+	aws_route53_resolver_endpoint;
+```
+
+### List of endpoints that allow inbound DNS queries
+
+```sql
+select
+	name,
+	id,
+	direction,
+	ip_address_count,
+	status
+from
+	aws_route53_resolver_endpoint
+where
+	direction = 'INBOUND';
+```
+
+### List of endpoints correctly configured and able to pass inbound or outbound DNS queries
+
+```sql
+select
+	name,
+	id,
+	direction,
+	ip_address_count,
+	status
+from
+	aws_route53_resolver_endpoint
+where
+	status = 'OPERATIONAL';
```

Comment:
we have status enum  `ACTION_NEEDED` , it would be better if we can show the list of endpoints where action is needed by the user.